src/workflows/dags/airnow.py

```diff
@@ -1,8 +1,7 @@
 from airflow.decorators import dag, task
 from airflow.utils.dates import days_ago
 from airqo_etl_utils.workflows_custom_utils import AirflowUtils
-from airqo_etl_utils.airnow_api import AirNowApi
```

Comment:
A quick one @MartinKalema , what happens to these removed imports?

- Are they not being utilised?
- Or were they already imported in another place? So you were removing a duplicate import?

Reply:
@Baalmart  They are not being utilised anywhere.